experimental/Schemes/ToricSchemes/NormalToricSchemes/Type.jl

```diff
@@ -0,0 +1,114 @@
+export ToricCoveredScheme
+
+
+@attributes mutable struct ToricCoveredScheme{BRT, CoveredSchemeType<:CoveredScheme} <: AbsCoveredScheme{BRT}
+  X::CoveredSchemeType
+  ntv::NormalToricVariety
+
+  function ToricCoveredScheme(ntv::NormalToricVariety)
+    F = fan(ntv)
+    C = maximal_cones(F)
+
+    antv = affine_open_covering(ntv)
+    rings = [PolynomialRing(coefficient_ring(antv[k]), ["x_$(i)_$(k)" for i in 1:ngens(base_ring(toric_ideal(antv[k])))], cached=false)[1] for k in 1:length(antv)]
+    patch_list = [ToricSpec(U, R=R) for (U, R) in zip(antv, rings)]
+    cov = Covering(patch_list)
+
+    for i in 1:(length(patch_list)-1)
+      X = patch_list[i]
+      CX = cone(X)
+      CXdual = dual_cone(X)
+      for j in i+1:length(patch_list)
+        Y = patch_list[j]
+        CY = cone(Y)
+        CYdual = dual_cone(Y)
+
+        # Now glue X and Y
+        
+        # Step 1: Find the cone of the intersection of X and Y.
+        facet = intersect(CX, CY)
+        # harvest the dual cone
+        dual_facet = polarize(facet)
+        if !(dim(facet) == dim(CX) - 1)
+          continue
+        end
```

Comment:
Is there a reason, why you polarize before testing without use of dual_facet whether you want to pursue this case or not?

Reply:
Probably yes. Frankly, I don't remember all details from the discussion with @lkastner from half a year ago.